.circleci/TestSplitting.yml

```diff
@@ -0,0 +1,53 @@
+version: 2.1
+orbs:
+  matlab: mathworks/matlab@1
+  win: circleci/windows@5
+
+# Define the execution environments for different operating systems
+executors:
+  linux:
+    machine:
+      image: default
+  macos:
+    macos:
+      xcode: 16.4.0
+  windows:
+    win/default
+
+jobs:
+  split-and-run-test:
+    parameters:
+      os:
+        type: executor
+    executor: << parameters.os >>
+    # Use parallelism to split tests across multiple containers
+    parallelism: 2
+    steps:
+      - checkout
+      
+      # Install MATLAB and required products
+      - matlab/install:
+          products: >
+            MATLAB_Compiler_SDK
+            MATLAB_Test
+      
+      # Builds mex file and Python package from MATLAB function
```

Comment:
Builds mex --> Build MEX

(Use Build instead of Builds for consistency with other comments.)